virt/virt-2-5-release-notes.adoc

```diff
@@ -9,14 +9,14 @@ toc::[]
 Red Hat {VirtProductName} {VirtVersion} is supported for use on {product-title} 4.6 clusters. {VirtProductName} enables you to bring traditional virtual machines (VMs) into {product-title} where they run alongside containers, and are managed as native Kubernetes objects.
 
 //Branding-Logo
-{VirtProductName} is represented by the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName}] logo.
+{VirtProductName} is represented by the image:Operator_Icon-OpenShift_Virtualization-5.png[{VirtProductName},40,40] logo.
 
 include::modules/virt-what-you-can-do-with-virt.adoc[leveloffset=+2]
 // This line is attached to the above `virt-what-you-can-do-with-virt` module.
 // It is included here in the assembly because of the xref ban.
 You can use {VirtProductName} with either the xref:../networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc#about-ovn-kubernetes[OVN-Kubernetes] or the xref:../networking/openshift_sdn/about-openshift-sdn.adoc#about-openshift-sdn[OpenShiftSDN] default Container Network Interface (CNI) network provider.
 //Commented out because for GA release we have special text that covers this info. But going forward, we will want to include the following:
-include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
+//include::modules/virt-supported-cluster-version.adoc[leveloffset=+2]
```

Comment:
This was causing a verbatim `:leveloffset: +2` to show up, and the heading order got screwy, so I assume based on the commented lines before here that this was also meant to be commented out.

Reply:
Fixed differently via https://github.com/openshift/openshift-docs/pull/26997